CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### 2.1.0 (Next)
 
+* [#58](https://github.com/acm19/aws-request-signing-apache-interceptor/issues/58): Test and fix Maven Central - [@acm19](https://github.com/acm19).
```

Comment:
You meant to say "Test and fix _the release_ to Maven Central'

Reply:
Changed.